sklearn/preprocessing/_label.py

```diff
@@ -141,28 +143,33 @@ def inverse_transform(self, y):
 
         Parameters
         ----------
-        y : ndarray of shape (n_samples,)
+        y : array-like of shape (n_samples,)
```

Comment:
In https://github.com/scikit-learn/scikit-learn/pull/27113/files#r1590992914 `ndarray` was changed to `array`. I'm not sure I have a strong opinion on which is better, but we should try and converge on one thing :)

Reply:
Yeah I saw array. Over here since the others were already array like I just used that. We can change to array.

Reply:
I think we should keep `array-like` for input arguments to be consistent with our guide lines:

https://scikit-learn.org/dev/developers/contributing.html#guidelines-for-writing-docstrings

For return values and fitted attributes (derived from the data), I think we should stick to `ndarray` to document the default behavior of scikit-learn with `array_api_dispatch=False` for the time being.

Reply:
Made the update.